test-cloud/test-cloud-common/src/main/java/org/kie/cloud/tests/common/runner/TestRunnerFeature.java

```diff
@@ -1,46 +0,0 @@
-package org.kie.cloud.tests.common.runner;
-
-import java.io.IOException;
-import java.util.Properties;
-
-import org.junit.rules.TestWatcher;
-import org.junit.runner.Description;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-public class TestRunnerFeature extends TestWatcher {
-	private static final Logger LOG = LoggerFactory.getLogger(TestRunnerFeature.class);
-	private static final String COMMON_PROPERTIES = "test.properties";
```

Comment:
Should be TestRunnerFeature really removed? If so, we can delete test.properties as well, right?

Reply:
No, the problem here was that we had the TestRunnerFeature duplicated in different packages (due to a wrong merge).